src/agent/onefuzz-task/src/tasks/fuzz/libfuzzer/generic.rs

```diff
@@ -37,7 +39,67 @@ impl common::LibFuzzerType for GenericLibFuzzer {
             config.common.machine_identity.clone(),
         ))
     }
+
+    async fn extra_setup(config: &common::Config<Self>) -> Result<()> {
```

Comment:
How does this loop for all the DLL files that might be in the directory?

Reply:
It's only needed for the entry executable